lib/internal/test_runner/runner.js

```diff
@@ -33,7 +33,7 @@ const { finished } = require('internal/streams/end-of-stream');
 const { resolve } = require('path');
 const { DefaultDeserializer, DefaultSerializer } = require('v8');
 // TODO(aduh95): switch to internal/readline/interface when backporting to Node.js 16.x is no longer a concern.
```

Comment:
I'm guessing you would want to remove this comment as well.

Reply:
yes thank you